services/create_portal_link_endpoint.go

```diff
@@ -0,0 +1,47 @@
+package services
+
+import (
+	"context"
+	"errors"
+	"net/http"
+
+	"github.com/frain-dev/convoy/cache"
+
+	"github.com/frain-dev/convoy/api/models"
+	"github.com/frain-dev/convoy/datastore"
+	"github.com/frain-dev/convoy/util"
+)
+
+type CreateEndpointPortalLinkService struct {
+	PortalLinkRepo datastore.PortalLinkRepository
+	EndpointRepo   datastore.EndpointRepository
+	Cache          cache.Cache
+	ProjectRepo    datastore.ProjectRepository
+
+	Project    *datastore.Project
+	Data       models.Endpoint
+	PortalLink *datastore.PortalLink
+}
+
+func (p *CreateEndpointPortalLinkService) Run(ctx context.Context) (*datastore.Endpoint, error) {
```

Comment:
This service isn't being used anywhere

Reply:
@Dotunj Yeah i noticed, i think it was removed temporarily.